partner_address_ok/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2024 Henrik Norlin
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    address_ok = fields.Boolean(
```

Comment:
I'm a bit perplexed.

It adds a boolean fields without adding any logic such as testing the validity of the address. We have something that exists in -> https://github.com/odoo/odoo/blob/17.0/addons/snailmail/models/snailmail_letter.py#L453

It could be achieved with tags on partners.
I would have prefered a field naming like `is_valid_postal_address` which is self explanatory.

Reply:
Thank you @leemannd for very good feedback!

Reply:
NB: another solution that was used in 12.0 was to add it into the type -> https://github.com/OCA/partner-contact/tree/12.0/partner_postal_address